api/qsharp/microsoft.quantum.canon.applyquantumfouriertransformbe.yml

```diff
@@ -3,11 +3,7 @@ uid: microsoft.quantum.canon.applyquantumfouriertransformbe
 name: ApplyQuantumFourierTransformBE
 type: operation
 namespace: Microsoft.Quantum.Canon
-summary: |-
-  > [!WARNING]
-  > Deprecated
-
-  This operation has been removed.
+summary: ApplyQuantumFourierTransformBE has been deprecated. Please use @"applyquantumfouriertransform(bigendianaslittleendian(qs))" instead.
```

Comment:
It looks like something went wrong with the deprecation summary here, in that `applyquantumfouriertransform(bigendianaslittleendian(qs))` isn't a valid UID.

Reply:
Ok, there should still be a valid link, so I suggest to take it in here. I'll file a task under the general clean-up deliverable.